app/main.py

```diff
@@ -2,0 +1,20 @@
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    issues_found = 0
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            issues_found += 1
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            issues_found += 1
+            masks_to_buy += 1
+    if masks_to_buy != 0:
+        return f"Friends should buy {masks_to_buy} masks"
+    if issues_found == 0:
```

Comment:
The condition `if issues_found == 0:` assumes that no issues mean all friends can go to the cafe. However, if a `VaccineError` is encountered, the function returns early, so this condition might not be reached as expected. Consider revising the logic to ensure all friends are evaluated before making a decision.